.maestro/testsV2/reusableFlows/features/auth/LoginFromProfile.yml

```
@@ -0,0 +1,14 @@
+appId: ${MAESTRO_APP_ID}
+---
+- assertVisible: 'Mon profil'
+- tapOn: 'Se connecter'
+
+- assertVisible: 'Connecte-toi'
+- tapOn: 'tonadresse@email.com'
+- inputText: ${MAESTRO_VALID_EMAIL}
+- tapOn: 'Ton mot de passe'
+- inputText: ${MAESTRO_PASSWORD}
+- tapOn: 'Se connecter'
+- tapOn: 'Se connecter'
```

Je pense qu'il faudrait mettre un commentaire (en anglais) a droit du deuxième tapOn pour expliquer pourquoi c'est doublé 
```suggestion
- tapOn: 'Se connecter'
- tapOn: 'Se connecter' // ...
```